client/src/App.vue

```diff
@@ -49,14 +49,24 @@ export default {
                   return Promise.reject(Text.EXPIRED_REFRESH_TOKEN)
                 }
                 return StoreHelper.fetchTokenData(token, apiUrl)
+              }).catch(err => {
+                if (
+                  ( err.response.status === 401  && 
+                    err.response.data.includes('The token has expired.') && 
```

Comment:
Using the expected error string here is fragile. If the message ever changes on the server this code will fail.  A less fragile approach needs two things.  First reduce the check to see if the work "expire" is in the message regardless of case. (eg. message.toLowerCase().includes('expired')) and second, add a comment to the server "the message MUST contain the word expired for clients to correctly handle a browser refresh when an expired refresh token is in the URL"